docs/Software Architecture Document/Component Interaction View/assets/app_launch.plantuml

```diff
@@ -0,0 +1,47 @@
+@startuml
```

Comment:
@LevchenkoS is it further used? did not find you reference this diagram at the page.